source/client.c

```diff
@@ -867,6 +930,8 @@ uint16_t aws_mqtt_client_connection_subscribe_multiple(
     uint16_t packet_id =
         mqtt_create_request(task_arg->connection, &s_subscribe_send, task_arg, &s_subscribe_complete, task_arg);
 
+    AWS_LOGF_DEBUG(AWS_LS_MQTT_CLIENT, "Starting subscribe %d", packet_id);
```

Comment:
topic?

Reply:
Can't without generating a big string as this is the multi-topic subscribe